### PR TITLE
[Doc Improvement][Deletion of Tab margin article][3781299]

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/cc66a1daea6a62acab43215fc6075c6b8a016241/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4c8786a-ba07-447a-9fad-fa9306647118) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc66a1daea6a62acab43215fc6075c6b8a016241/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c8899fc4-82d2-4e75-ba04-894aba1bdb1f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc66a1daea6a62acab43215fc6075c6b8a016241/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 89 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06dc6144-7b35-47f0-98cf-017be497019f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc66a1daea6a62acab43215fc6075c6b8a016241/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac2f5be6-df78-4b07-9d51-586002ea17d7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4ff492a55b061e9b517ef2aaec0f92e779ad1592/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/45b9218c-8d94-4c49-b336-7fb94e8e8226) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ff492a55b061e9b517ef2aaec0f92e779ad1592/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/144d83dc-5070-4568-84d0-cb28a7b370b4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ff492a55b061e9b517ef2aaec0f92e779ad1592/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/03ef13b2-e95a-49b5-9893-eeb4b9b5b88f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4ff492a55b061e9b517ef2aaec0f92e779ad1592/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d754dfda-1e99-4e60-a562-1e0d6f6d0d28) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1d53ff014c942c7f2bfb250f0871a5b34b63eed8/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/22eac26e-068a-442f-bd3b-5a5a5016e7ac) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d53ff014c942c7f2bfb250f0871a5b34b63eed8/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/61392f79-71b1-43a3-bea9-693a7f3b3f2d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d53ff014c942c7f2bfb250f0871a5b34b63eed8/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32919ecc-671a-4170-a6a8-740771a15654) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d53ff014c942c7f2bfb250f0871a5b34b63eed8/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e85a995f-e788-44ee-86f5-8b4bde1dfb47) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/whats-new.md [hunk 1]
```diff
@@ -482,6 +481,8 @@ Discover Microsoft Teams platform features that are deprecated. You can now get
 
 Teams platform features that aren't available.
 
+* ***April 30, 2024***: Tab margin experience is now deprecated for all types of tabs.
+
 * ***April 12, 2024***: The `packageName` property is deprecated as part of manifest v1.17.
```
Comment (v-npaladugu) [inline msteams-platform/whats-new.md:486]: app manifest

File: msteams-platform/whats-new.md [hunk 2]
```diff
@@ -482,6 +481,8 @@ Discover Microsoft Teams platform features that are deprecated. You can now get
 
 Teams platform features that aren't available.
 
+* ***April 30, 2024***: Tab margin experience is now deprecated for all types of tabs.
```
Comment (v-npaladugu) [inline msteams-platform/whats-new.md:484]: In tab experiences, tab margins are deprecated.

**Review by v-npaladugu — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0175cecc5367a24142dab1fff4a567961fdab857/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d13def38-dfa3-4f12-bdf1-8bbebdd61f40) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0175cecc5367a24142dab1fff4a567961fdab857/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f353de62-7869-42fd-baab-a3cc80026b57) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0175cecc5367a24142dab1fff4a567961fdab857/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a43aa378-e3fc-4de7-8dcd-98a4d719ceda) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0175cecc5367a24142dab1fff4a567961fdab857/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e98338e-18fc-49f3-8374-b198173ff1ae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/whats-new.md (hunk 2 above)
Comment (v-ganr) [reply msteams-platform/whats-new.md:484]: Updated.

File: msteams-platform/whats-new.md (hunk 1 above)
Comment (v-ganr) [reply msteams-platform/whats-new.md:486]: Existing error, fixed.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a5bd42e677956bc4dfe5783070d9e766d814403e/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/04908b66-0711-4af7-87e1-cd5966d60d1a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a5bd42e677956bc4dfe5783070d9e766d814403e/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed579c2d-4425-4bfe-a4a1-6f2db2bf8dfb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a5bd42e677956bc4dfe5783070d9e766d814403e/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5cccb8c-d028-4358-b84f-4023109ab353) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a5bd42e677956bc4dfe5783070d9e766d814403e/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4e73b8d5-58f8-4960-857c-3007312d1628) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ec03e3a41bc91f2df159c94d9c29efa21629f384/msteams-platform%2FTOC.yml) | 96 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8347badf-5f31-4c7d-aaea-1eadce0d9277) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/design/tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ec03e3a41bc91f2df159c94d9c29efa21629f384/msteams-platform%2Ftabs%2Fdesign%2Ftabs.md) | 91 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa7c4a4b-3ec6-471a-ade4-18f15922fd92) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/conversational-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ec03e3a41bc91f2df159c94d9c29efa21629f384/msteams-platform%2Ftabs%2Fhow-to%2Fconversational-tabs.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/407fe7a4-6f6a-4daa-97d6-4979ae8be48e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ec03e3a41bc91f2df159c94d9c29efa21629f384/msteams-platform%2Fwhats-new.md) | 91 | 77 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff91b9d7-339c-4fd7-bb2f-9156d9380fbb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-npaladugu — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**